trigger/project_triggers.go

```diff
@@ -53,6 +52,9 @@ func TriggerDownstreamVersion(ctx context.Context, args ProcessorArgs) (*model.V
 		return nil, errors.Wrap(err, "getting Evergreen settings")
 	}
 	var versionID string
+	// For build and task level triggers, there is a source version from which we can extract a project ID.
+	// For push triggers, since there is no source version and the ProjectID field gets populated from
```

Comment:
this isn't quite a real sentence, could you tweak? Maybe like " Since there is no source version for push triggers,  the ProjectID field gets populated from..."